### PR TITLE
[IMP] hr_timesheet_sheet_attendance: add Attendance to summary view

### PR DESCRIPTION
**Pull request by tschanzt**
Adds a line for the attendance of each day in the summary view on the timesheet_sheet
Screenshot from 2019-03-14 17-16-29

### CODE REVIEW

**Review by leemannd — APPROVED** [review]
Functional testing only

File: hr_timesheet_sheet_attendance/models/hr_timesheet_sheet.py [hunk 1]
```diff
@@ -83,6 +83,43 @@ def check_employee_attendance_state(self):
...
                 "not contain an equal number of sign ins and sign outs."))
 
+    @api.multi
+    def get_attendance_by_day(self):
+        self.ensure_one()
+        result = []
+        date_end = fields.Date.from_string(self.date_end)
+        date_start = fields.Date.from_string(self.date_start)
+        date_range = (date_end - date_start).days
+        for delta in range(0, date_range + 1):
```
Comment (p-tombez) [inline hr_timesheet_sheet_attendance/models/hr_timesheet_sheet.py]: ```result = [0] * (date_range + 1)```

File: hr_timesheet_sheet_attendance/tests/test_attendance_by_day.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            employee=self.employee,
+            checkIn=fields.Datetime.from_string('2019-04-01 10:00:00'),
+            checkOut=fields.Datetime.from_string('2019-04-01 15:00:00')
+        )
+        res = self.timesheet.get_attendance_by_day()
+        self.assertEqual(res[0], 5.0)
+        self.assertEqual(res[1], 0.0)
+        self.assertEqual(res[2], 0.0)
+        self.assertEqual(res[3], 0.0)
+        self.assertEqual(res[4], 0.0)
```
Comment (p-tombez) [inline hr_timesheet_sheet_attendance/tests/test_attendance_by_day.py]: ```self.assertEqual``` also works with lists, so you can simplify all these calls with only one : 
```
self.assertEqual(
    res, [5.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
)
```

File: hr_timesheet_sheet_attendance/tests/test_attendance_by_day.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
+from odoo.addons.hr_timesheet_sheet_attendance.tests.\
+    hr_timesheet_sheet_test_cases import HrTimesheetTestCases
+from odoo import fields
+
+
+class TestAttendanceByDay(HrTimesheetTestCases):
```
Comment (p-tombez) [inline hr_timesheet_sheet_attendance/tests/test_attendance_by_day.py:6]: Can you add a test case for attendances spanning on 2 days with a gap ?
e.g. : 
2019-04-02 19:00:00 to 2019-04-03 05:00:00
2019-04-03 19:00:00 to 2019-04-04 05:00:00

Comment (p-tombez) [inline hr_timesheet_sheet_attendance/tests/test_attendance_by_day.py:6]: There is a lot of duplicated code, maybe you could try to write a generic method to which you can pass a list of intervals and the expected result

**Review by p-tombez — COMMENTED** [review]
Base LG, please consider these improvements

**Comment by tschanzt** [comment]
@p-tombez you want to have another look?

**Review by p-tombez — COMMENTED** [review]
Overall LG, but Travis is :red_circle:

**Comment by leemannd** [comment]
@tschanzt Hello, your test will not pass as Odoo calulates them this way:
test_attendencies

**Review by p-tombez — APPROVED** [review]

**Pull request merged**